tools/docker/builder/Dockerfile

```diff
@@ -12,7 +12,8 @@ FROM $image
 RUN apt-get update && apt-get install -y \
 	curl \
 	gcc \
-	cmake \
+        cmake \
+        clang-format \
```

Comment:
I realize it's already not alphabetically ordered, but please put clang-format before cmake :-)

For indentation: I think it would be better to convert the file to (4) spaces like we use in the rest of the project. But only do that if you have time for it :-)